ui/src/components/NewExperiment/Stepper/Target/IO.tsx

```diff
@@ -0,0 +1,120 @@
+import { InputAdornment, MenuItem } from '@material-ui/core'
+import { SelectField, TextField } from 'components/FormField'
+
+import AdvancedOptions from 'components/AdvancedOptions'
+import React from 'react'
+import { StepperFormTargetProps } from 'components/NewExperiment/types'
+import { toTitleCase } from 'lib/utils'
+
+const actions = ['delay', 'errno', 'mixed']
+const methods = [
+  'open',
+  'read',
+  'write',
+  'mkdir',
+  'rmdir',
+  'opendir',
+  'fsync',
+  'flush',
+  'release',
+  'truncate',
+  'getattr',
+  'chown',
+  'chmod',
+  'utimens',
+  'allocate',
+  'getlk',
+  'setlk',
+  'setlkw',
+  'statfs',
+  'readlink',
+  'symlink',
+  'create',
+  'access',
+  'link',
+  'mknod',
+  'rename',
+  'unlink',
+  'getxattr',
+  'listxattr',
+  'removexattr',
+  'setxattr',
+]
+
+export default function IO(props: StepperFormTargetProps) {
+  const { values, handleActionChange } = props
+
+  return (
+    <>
+      <SelectField
+        id="target.io_chaos.action"
+        name="target.io_chaos.action"
+        label="Action"
+        helperText="Please select an action"
+        onChange={handleActionChange}
+      >
+        {actions.map((option: string) => (
+          <MenuItem key={option} value={option}>
+            {toTitleCase(option)}
+          </MenuItem>
+        ))}
+      </SelectField>
+
+      {values.target.io_chaos.action !== '' && (
+        <AdvancedOptions>
+          {values.target.io_chaos.action === ('delay' || 'mixed') && (
```

Comment:
I am not really sure but this seems like a bug. Should it be `(values.target.io_chaos.action === 'delay' || values.target.io_chaos.action === 'mixed')` ?

Reply:
Ah...a place that shouldn’t be wrong, I’ll fix it right away!